packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

```diff
@@ -33,8 +43,7 @@ function isHook(node) {
     isHook(node.property)
   ) {
     const obj = node.object;
-    const isPascalCaseNameSpace = /^[A-Z].*/;
-    return obj.type === 'Identifier' && isPascalCaseNameSpace.test(obj.name);
+    return obj.type === 'Identifier' && !nonHookNamespaces.includes(obj.name);
```

Comment:
- I don't think the eslint rule should care if an object is PascalCased or not to decide whether the member is a hook or not. 
- I don't know if having a list of allow-listed namespaces is acceptable, nor if such an alternative then should be configurable.